src/main/kotlin/com/cognifide/gradle/aem/common/instance/Instance.kt

```diff
@@ -143,6 +143,7 @@ open class Instance(val aem: AemExtension, val name: String) {
 
     val runningModes get() = slingSettings["Run_Modes"]
         ?.removeSurrounding("[", "]")
+        ?.removePrefix(",")
```

Comment:
why?

Reply:
always think about the most defensive code you could have here
how about `.filter { it.isNotBlank() }`

Reply:
because it returns a string started with a comma, I couldn't go deeper to slingSettings

Reply:
but the issue was more about the reason, why somewhere there is that comma only
not about the GAP itself; fix should be done on both sides if possible; I mean on the AEM setup side might be an issue causing that comma